pages/[id]/FeatureFlags.js

```diff
@@ -0,0 +1,197 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import Page from '../../components/page';
+import Card, {
+  CardHeader,
+  CardButtonContainer,
+  CardTitle,
+  CardContent,
+} from '../../components/card';
+import Button from '../../components/button';
+import { COLOR, EDIT } from '../../constants';
+import {
+  getTimestamp,
+  subscribeToFlags,
+  formatDate,
+  updateFlags,
+  getHackathonPaths,
+  getHackathons,
+} from '../../utility/firebase';
+import { useAuth } from '../../utility/auth';
+
+const CancelText = styled.p`
+  border-bottom: 2px solid ${COLOR.BLACK};
+  margin: 0px;
+`;
+
+const CancelButton = styled.button`
+  font-size: 16px;
+  cursor: pointer;
+  border-bottom: 2px solid ${COLOR.BLACK};
+  margin-left: 675px;
+  margin-right: 40px;
+  border: none;
+  outline: none;
+  background-color: ${COLOR.BACKGROUND};
+`;
+
+const FeatureFlagsContainer = styled.div`
+  display: flex;
+  width: 40%;
+  border: 2px solid lightgray;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  justify-content: space-between;
+  background-color: #ededed;
```

Comment:
As per my prev suggestion, I think this would look more consistent and native to our theme
```suggestion
  background-color: ${COLOR.WHITE};
```
![Screen Shot 2020-09-17 at 11 08 03 PM](https://user-images.githubusercontent.com/38872354/93562429-e09c9900-f93a-11ea-8fea-45a7e27439bd.jpg)